test/functional/feature_assumeutxo.py

```diff
@@ -82,7 +82,7 @@ def test_invalid_snapshot_scenarios(self, valid_snapshot_path):
             f.write(bytes.fromhex(bad_snapshot_block_hash)[::-1] + valid_snapshot_contents[32:])
 
         expected_log = f"assumeutxo height in snapshot metadata not recognized ({bad_snapshot_height}) - refusing to load snapshot"
-        with self.nodes[1].assert_debug_log(expected_log):
+        with self.nodes[1].assert_debug_log([expected_log]):
```

Comment:
Note this is also in #28639.

Reply:
After rebase, that change is now not included anymore. Mentioned in the PR description that the first of the three instances is already fixed by https://github.com/bitcoin/bitcoin/pull/28639.